docs/user_guide/HOWTO/Changing_Puppet_Masters.rst

```diff
@@ -1,39 +1,108 @@
-HOWTO Change Puppet Masters
+HOWTO Change Puppet Servers
 ===========================
 
-It may be necessary to change the Puppet Master. To point a particular
-client to a new Puppet Master, follow the steps in the sections below.
+It may be necessary to change the Puppet Server. To point a particular
+client to a new Puppet Server, follow the steps in the sections below.
 
-On the Client
--------------
+.. NOTE::
 
-Enter the following changes into the */etc/puppetlabs/puppet/puppet.conf* file.
+   All commands in this section should be run as the ``root`` user.
 
-Code Changes on Client to Switch Puppet Masters
+On the Old Puppet Server
+------------------------
+
+Collect the Client's Server-Side Artifacts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Until SIMP implements a shared Puppet data store (expected 2017-Q2), you will
+need to manually copy some artifacts from the old server to the new server
+
+To do this, run:
+
+.. code-block:: shell
+
+   $ find `puppet config --section master print vardir`/simp` -name "*<client-fqdn>*" -exec tar --selinux --xattrs -rpvf <client-fqdn>_transfer.tar {} \;
```

Comment:
I think the first bit should be find find `puppet config --section master print vardir`/simp